scene/3d/sprite_3d.cpp

```diff
@@ -34,15 +34,13 @@
 #include "scene/scene_string_names.h"
 
 Color SpriteBase3D::_get_color_accum() {
-	if (!color_dirty) {
```

Comment:
Both `sprite_3d.cpp` and `sprite_3d.h` have a bunch of code style changes, like removing braces, changing `nullptr` to `NULL` and removing initialization from the header. I guess you used an old version of the file as a base?